src/main/resources/application.properties

```
@@ -141,6 +141,10 @@ search.elastic.hosts=${ELASTIC_SEARCH_HOSTS:http://localhost:9200}
 search.elastic.data.hosts=${ELASTIC_SEARCH_DATA_NODES_HOSTS:"http://localhost:9200"}
 search.blacklist=${ELASTIC_SEARCH_BLACKLIST:query_string}
 search.cases.index.name.format=${ELASTIC_SEARCH_CASE_INDEX_NAME_FORMAT:%s_cases}
+#regex with a group used to identify and extract the case id from the index name
+search.cases.index.name.case-type-id.group=(.+?)(_cases.*)
```

```suggestion
search.cases.index.name.case-type-id.group=(.+)(_cases.*)
```
Think we may need to let it be greedy? At the moment if any services were to use a case type such as `ENGLISH_CASES` then the index would be `english_cases_cases...` and the first capturing group would only be `english`.

---

ehhhhh ... lest discuss tomorrow this one

---

done